impl/src/main/java/org/jboss/weld/config/WeldConfiguration.java

```diff
@@ -123,8 +123,8 @@ public WeldConfiguration(ServiceRegistry services, Deployment deployment) {
         this.proxyDumpFilePath = initProxyDumpFilePath();
         this.proxyIgnoreFinalMethodsPattern = initProxyIgnoreFinalMethodsPattern();
         StringJoiner logOutputBuilder = new StringJoiner(", ", "{", "}");
-        for (ConfigurationKey key : properties.keySet()) {
-            logOutputBuilder.add(key.get() + "=" + properties.get(key));
+        for (Entry<ConfigurationKey, Object> entry : properties.entrySet()) {
+            logOutputBuilder.add(entry.getKey().get() + "=" + properties.get(entry.getKey()));
```

Comment:
`properties.get(entry.getKey())` should be `entry.getValue()`, otherwise I see no improvement but just an attempt to make spotbugs happy ;-)